backend/app/usecases/bot.py

```diff
@@ -131,6 +132,17 @@ def create_new_bot(user_id: str, bot_input: BotInput) -> BotOutput:
                 chunk_size=chunk_size,
                 chunk_overlap=chunk_overlap,
             ),
+            generation_config=(
```

Comment:
Create dynamo item with default values if null. Refer the very above code `chunk_size` and `chunk_overlap`, which are for embedding params (They are defined outside of `store_bot`, but inside-definition should be readable. If you agree that, could you move them (chunk_*) to inside?).

Reply:
For generation_config, the request is not nullable, so we do not need to set the default value.